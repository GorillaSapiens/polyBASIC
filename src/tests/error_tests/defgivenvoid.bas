DEF a(x) x + 1
b = a(void)
