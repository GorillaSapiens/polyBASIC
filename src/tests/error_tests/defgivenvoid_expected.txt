ERROR: @1:10, UNKNOWN OP IN PARAMETER SUBSTITUTION ❮VOID❯
